app/src/main/res/values/strings.xml

```diff
@@ -178,14 +178,14 @@
 
     <!-- Hint Strings -->
     <!-- Hint text displayed when the overlay is open.
-     %IMAGE will be replaced with the image of a hardware back button.
+     $IMAGE will be replaced with the image of a hardware back button.
      When this button is pressed, the overlay will close and the user will be returned to the browser. -->
-    <string name="hint_press_back_to_close_overlay">Press %IMAGE to return</string>
+    <string name="hint_press_back_to_close_overlay">Press $IMAGE to return</string>
     <string name="hint_press_back_to_close_overlay_a11y">Press the back button to return</string>
     <!-- Hint text displayed when the browser is open.
-     %IMAGE will be replaced with the image of a hardware menu button.
+     $IMAGE will be replaced with the image of a hardware menu button.
      When this button is pressed, the overlay will open and the browser will be hidden underneath it -->
-    <string name="hint_press_menu_to_open_overlay">Press %IMAGE for Firefox TV Home, search, settings, and more</string>
-    <string name="hint_press_menu_to_open_overlay_a11y">Press the menu button for Firefox TV Home, search, settings, and more</string>
+    <string name="hint_press_menu_to_open_overlay">Press $IMAGE for %1$s Home, search, settings, and more</string>
```

Comment:
One nit, in the localization notes for these two strings also include what the %1$s will be replaced by.